### PR TITLE
Feature/ci docker image

### PR DESCRIPTION
**Pull request by forgondolin**
Hello, something like this? 
Any testing?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  build:
+    docker:
+      - image: cibuilds/docker:19.03
+    steps:
+      - checkout
+      - setup_docker_php
+      - run:
+          name: "Build Docker Image"
+          command: docker build --file Dockerfile
+      - deploy:
```
Comment (pschoffer) [inline .circleci/config.yml:13]: the Deploy step is not needed I think. I have docker hub integration in place that should listen to new commits in master and do that thing on its own, or?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+version: 2.0
+
+jobs:
+  build:
+    docker:
+      - image: cibuilds/docker:19.03
+    steps:
+      - checkout
+      - setup_docker_php
```
Comment (pschoffer) [inline .circleci/config.yml:9]: What does this do? I guess we don't need `php` just docker, which I guess we got already in the image you have used above.

**Review by pschoffer — CHANGES_REQUESTED** [review]
Great stuff.

I have configured the circleCI integration, but for that I merged some hello word config, which cause conflicts here, sorry about that. Please rebase or merge master to this branch and resolve those.

However I like the simpler approach you had in here rather than the `orbs` and `workflow` stuff in the hello word example. So  I would suggest going more in your direction.

One last thing, this should only deal with the circleci not the plugin move which is included in this pr by mistake probably.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      - run:
+          name: "Build Docker Image"
+          command: docker build --file Dockerfile
```
Comment (pschoffer) [inline .circleci/config.yml:12]: Yep this line is exactly what I had in mind :) Thanks

File: .circleci/config.yml (hunk 2 above)
Comment (forgondolin) [reply .circleci/config.yml:9]: Yeah, actually I forgot to remove the setup step, my bad

**Comment by forgondolin** [comment]
Im closing this PR to reopen tomorrow, lol

**Pull request closed without merging**